### PR TITLE
Update persistent_storage_nfs.adoc

### PR DESCRIPTION
**Pull request by erinboyd**
Create note to newer settings for NFS exports

### CODE REVIEW

**Comment by erinboyd** [comment]
@adellape please review

File: admin_guide/persistent_storage_nfs.adoc [hunk 1]
```diff
@@ -167,3 +167,8 @@ The export definition above allows arbitrary network clients to mount this
 volume. Exports can be restricted to a range of IP addresses for hosts that will
 access the volume. See `man exports` for more information.
 ====
+ 
+[ADMONITION]
```
Comment (adellape) [inline admin_guide/persistent_storage_nfs.adoc]: s/ADMONITION/IMPORTANT/

File: admin_guide/persistent_storage_nfs.adoc [hunk 2]
```diff
@@ -167,3 +167,8 @@ The export definition above allows arbitrary network clients to mount this
...
+====
+For OpenShift 3.1 and greater the export values have changed. Please follow this guide to ensure proper security for NFS link:https://docs.openshift.com/enterprise/3.1/install_config/persistent_storage/persistent_storage_nfs.html[NFS setup for OpenShift 3.1]
```
Comment (adellape) [inline admin_guide/persistent_storage_nfs.adoc]: Starting in OpenShift Enterprise 3.1, the export values have changed. See the https://docs.openshift.com/enterprise/3.1/install_config/persistent_storage/persistent_storage_nfs.html[OpenShift Enterprise 3.1 documentation] for instructions on ensuring proper security for NFS in 3.1.

**Comment by erinboyd** [comment]
@adellape updated doc per your comments

**Comment by adellape** [comment]
@erinboyd Thanks. Can you squash? Will merge after.

**Pull request closed without merging**